### ISSUE TITLE
OCR can't recognize this subs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [quemoni/subtitleedit#28]
```
What steps will reproduce the problem?
1.Import a .sub file (which I attach)
2.Adjust image palette in ocr dialog (optional)
3.Press "Start OCR"

What is the expected output? What do you see instead?
I expect that OCR recognize first line but never happen, no matter which custom 
color I use in image palette

What version of the product are you using? On what operating system?
Version 3.1, OS Windows 7 Ultimate X64

Please provide any additional information below.
I think this happen because sub image have 4 different levels (not 3), 1: inner 
peach color is text, 2: thin dashed dark gray is first border, 3: heavy light 
gray is second border, and 4: outside peach color is background. 
I set custom colors in image palette in order to see clearly subtitles, but the 
info that is sent to tesseract is wrong (because what I said before) and the 
OCR can't get right result. I attach subtitles.

```

Original issue reported on code.google.com by `ecoord...@gmail.com` on 4 May 2011 at 6:19

Attachments:
- [subs_sample.7z](https://storage.googleapis.com/google-code-attachments/subtitleedit/issue-28/comment-0/subs_sample.7z)

**Comment by GoogleCodeExporter** [comment]
```
Hi, try to use white for emphasis1 (second last color) and make all other 
colors transparent.

Yes, sub pictures have 4 colors, but the first color is background color 
(should be anyway). I can add background color to custom colors... but I don't 
think it will help.

Yes, Tesseract is a bit picky about image colors...
```

Original comment by `nikse.dk@gmail.com` on 4 May 2011 at 2:11

**Comment by GoogleCodeExporter** [comment]
```
I did try all custom colors I could think and in all this cases in subedit 
dialog text looks really clear; I set background color transparent and first 
border (dashed dark gray) too, or the first border with same color of text, but 
nothing worked. The problem is the second border (thick light gray, may be is 
hard to see in image because is very light), this is the problem and I guess 
when subedit send this info to Tesseract, the OCR can't analyze correctly this 
data. If I set background transparent or black for better contrast, and set 
text color with same color of second border, the resulting text is so thick 
that OCR can't read it.
With sub-file I sent you, subedit detects 3 colors: background, first border 
and text; and only allow me to set this 3 colors, but not second border. I 
think if custom colors of subedit allow me set more that 3 color, maybe I can 
set the second border (light gray) transparent and then OCR will read correctly 
the image.
```

Original comment by `ecoord...@gmail.com` on 6 May 2011 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
OK, this was due a few shortcomings in the dvd sub picture decoder + too 
aggressive cropping - this is fixed in SE 3.2 final.
```

Original comment by `nikse.dk@gmail.com` on 18 Sep 2011 at 12:50
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This is great. Thanks a lot.
```

Original comment by `ecoord...@gmail.com` on 19 Sep 2011 at 12:19
